### ISSUE TITLE
Pure ML Idea

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/auto_ml#202]
- Use the difference between two models (one trained on `median_absolute_error` and another trained on `RMSE`) to estimate uncertainty
- Unsupervised learning before feature_learning, or before doing deep learning generally
- More stacking and blending, since we've already got fl_data separated out
- Add regularization back into deep learning
- Try dropout in deep learning

**Comment by ClimbsRocks** [comment]
Return some measure of confidence/uncertainty
- can be number of samples in node (would be cool!)
- variation of those samples
- train a model to come in the pipeline after our finalmodelatc. that model will try to predict the likelihood of the previous model being off by some percent or some amount. 
- look into varying each column by a random amount up to one standard deviation, and getting a series of 10 predictions based on all those random permutations. then, average those predictions together for our final prediction. then, we can also return the variance of these predictions as an estimate of certainty.

**Comment by ClimbsRocks** [comment]
batch normalization! didn't realize it was implemented so easily in keras
